src/app/components/App.jsx

```diff
@@ -0,0 +1,17 @@
+'use strict';
+
+var React =  require('react');
+
+var App = React.createClass({
+
+  render: function () {
+
+    return (
+      <div>this is something different</div>
+    );
+  }
+
+});
```

Comment:
If we are using this syntax, should we use "import React from 'react';" when requiring a module? @jenriquejr @eespinal